src/modules/Kdump.rb

```diff
@@ -385,10 +385,24 @@ def ReadKdumpSettings
     #
     # @return [Boolean] whether successful
     def update_initrd
+      # For CaaSP we need an explicit initrd rebuild before the
+      # first boot, when the root filesystem becomes read only.
+      rebuild_cmd = "/usr/sbin/tu-rebuild-kdump-initrd"
+      # part of transactional-update.rpm
+      update_initrd_with("if test -x #{rebuild_cmd}; then #{rebuild_cmd}; fi")
```

Comment:
For the installation, the code returns early enough.
For a regular SLES running system, transactional-update.rpm will not be present.